docs/_documentations/troubleshooting.md

```diff
@@ -390,6 +390,33 @@ These steps reproduce the issue:
 
 **Workaround** Run the `Attach Debugger` action manually.
 
+<!--
+Codewind version: 0.6.0
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/1061
+-->
+## Appsody binds fail
+
+If an Appsody respository is already added in your local Appsody CLI, and if you try to add the Appsody repository again with a different name or ID, the Appsody bind might fail in the following scenario:
```

Comment:
I think we'd should remove "and if you try to add... name or ID" part.  Reason is when using add via template manager, Codewind generates an ID under the covers so users have no control over this.